internal/interpreter/dynamic_item_tree.rs

```diff
@@ -1341,6 +1343,7 @@ pub(crate) fn generate_item_tree<'id>(
         compiled_globals,
         change_trackers,
         timers,
+        popup_id: std::cell::Cell::from(0),
```

Comment:
I believe this might need to be a map from the (compile-time) popup id to the (runtime) popup id.

Reply:
Fixed. I also added a test that should catch this.